.gitignore

```diff
@@ -1 +1,6 @@
-!template.txt
\ No newline at end of file
+!template.txt
+venv
+vonv
+vinv
+.*
+additional_config.json
```

Comment:
You have this in the gitignore but also committing the file?

Reply:
This was just so the internet can see the template but at the same time I won't be pushing my template in future commits. Does it make sense?